internal/endorser/endorser.go

```diff
@@ -101,10 +116,10 @@ func loadAndVerifyProvenances(referenceValues *common.ReferenceValues, provenanc
 
 // verifyProvenances verifies the given list of provenances. An error is returned if not.
 // TODO(b/222440937): Document any additional checks.
```

Comment:
Can the documentation be updated now? (or perhaps it could be done in a separate PR :))